docs/content/concepts.mdx

```diff
@@ -8,9 +8,9 @@ Learn about Dagster's core concepts and how to use them in your data platform.
 
 ---
 
-## Software-defined assets
+## Software-defined Assets
 
-An asset is an object in persistent storage, such as a table, file, or persisted machine learning model. A software-defined asset is a Dagster object that couples an asset to the function and upstream assets used to produce its contents.
+An asset is an object in persistent storage, such as a table, file, or persisted machine learning model. A Software-defined Asset is a Dagster object that couples an asset to the function and upstream assets used to produce its contents.
 
 <ArticleList>
```

Comment:
Looks like the graph backed asset page never got added to this list, could you add it as part of this PR?